securitas.cfg.default

```diff
@@ -25,4 +25,7 @@ SESSION_COOKIE_SECURE = False
 PASSWORD_RESET_EXPIRATION = 10
 
 # Email sender
-MAIL_FROM = "Securitas <securitas@example.com>"
\ No newline at end of file
+MAIL_FROM = "Securitas <securitas@example.com>"
+
+# Set to False for prod
+TESTING = True
```

Comment:
Actually, since we're not in a testing environment and setting `TESTING` to `True` can have other side-effects, I would prefer if we set `MAIL_SUPPRESS_SEND` to `True` (which has the same effect for Flask-Mail [according to the doc](https://flask-mail.readthedocs.io/en/latest/#unit-tests-and-suppressing-emails))